Makefile

```diff
@@ -35,7 +35,7 @@ tests-doctests:
 
 .PHONY: tests-lib
 tests-lib:
-	pytest --durations=10 $(pytest_flags) --doctest-modules hathor --cov-fail-under=85 $(tests_lib)
+	pytest --durations=10 $(pytest_flags) --doctest-modules hathor --cov-fail-under=83 $(tests_lib)
```

Comment:
I can't disagree. I've no excuses. I'll add tests to the new block templates methods added to `HathorManager` and to `hathor.mining`. But I'll leave `hathor.merged_mining` alone, it's going to be removed from hathor-core anytime now, and it will benefit of doing tests like on the tx-mining-service, which require Python 3.8.

Reply:
Ok. I'm more concerned with testing methods like `get_parent_txs` and `get_block_templates`. Shouldn't be

Reply:
Added some tests. Can you take a look?

Reply:
Looks ok now..